public/locales/ko/Share.json

```
@@ -0,0 +1,6 @@
+{
+  "share_button": "X로 공유하기",
+  "find_all": "'샬레 총결산 with 총학생회 재고관리 계산기'를 사용하여 {{openPanelCount}}칸을 열고 {{allItemCount}}개의 모든 아이템을 찾았습니다!",
+  "find_some": "'샬레 총결산 with 총학생회 재고관리 계산기'를 사용하여 {{openPanelCount}}칸을 열고 {{allItemCount}}개 중 {{allItemCount}}개의 아이템을 찾았습니다!",
```

The second `{{allItemCount}}` should be `{{foundItemCount}}`, since this message is intended for the case where not all items were found.

```suggestion
  "find_some": "'샬레 총결산 with 총학생회 재고관리 계산기'를 사용하여 {{openPanelCount}}칸을 열고 {{allItemCount}}개 중 {{foundItemCount}}개의 아이템을 찾았습니다!",
```

---

I was confused with `find_all` and `find_some`. Fixed!